src/implementations/twig/components/card/card.story.js

```diff
@@ -18,123 +9,187 @@ import dataCardTile from '@ecl/specs-component-card/demo/data--tile';
 import card from './card.html.twig';
 import notes from './README.md';
 
-const iconsList = [];
-generalIcons.forEach((icon) => {
-  iconsList.push(icon);
-});
+const getArgTypes = (data) => {
+  const argTypes = {};
+  argTypes.title = {
+    type: { name: 'string', required: true },
+    defaultValue: data.card.title.label,
+    description: 'The card title',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+  };
+  argTypes.description = {
+    type: 'string',
+    defaultValue: data.card.description,
+    description: 'The card description',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+  };
+  if (data.card.meta) {
+    argTypes.meta = {
+      name: 'meta (comma separated)',
+      type: { name: 'array' },
+      defaultValue: data.card.meta,
+      description: 'The card meta',
+      table: {
+        type: { summary: 'array' },
```

Comment:
it may be better to put the "comma separated" information here instead of in the name (same for other controls)